spotinst/credential_aws/fields_spotinst_credential_aws.go

```diff
@@ -0,0 +1,82 @@
+package credential_aws
+
```

Comment:
change file name

Reply:
updated file name to fields_spotinst_credentials_aws.go